material-github.user.css

```diff
@@ -1181,9 +1181,9 @@
 	input[type="checkbox"]:hover:disabled,
 	input[type="checkbox"]:hover:active:disabled {
 		-webkit-appearance: none !important;
-		background-color: #25272a !important;
-		border: 1px solid #484848 !important;
-		color: #eee !important;
+		background-color: rgba(37,39,42) !important; 
+		border: 1px solid rgba(72,72,72) !important;
+		color: rgba(238,238,238) !important;

```

Comment:
Sorry, that's not what I meant. You should instead use `var(--bg3)` (replace 3 with the number that applies) when it's possible, or `rgba(255, 255, 255, .7)` (replace 7 with the number that applies) in certain cases like text color sometimes. 
This is required to make the colors variable, and work with all different theme variations (Ocean, Darker, Palenight, or Custom)

Reply:
Oh, Let me try again.